### PR TITLE
Gh11501 process export csv

### PR DESCRIPTION
**Pull request by metas-rc**

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+
+// FIXME: remote this class
+@Deprecated
+public class ExportToExcelProcess extends ExportToSpreadsheetProcess
```
Comment (teosarca) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java:30]: pls remove it

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5598410_sys_gh11501_AD_Process_csv_Columns.sql [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+
+
+
+
+
+
+
+UPDATE AD_Process SET spreadsheetformat = 'xls'
+WHERE spreadsheetformat IS NULL;
```
Comment (teosarca) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5598410_sys_gh11501_AD_Process_csv_Columns.sql:175]: AND type is Excel.

File: backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java (hunk 1 above)
Comment (teosarca) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java:30]: also provide a migration script which updates AD_Process and sets Classname to ExportToSpreadsheetProcess where the classname is ExportToExcelProcess

Comment (teosarca) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java:30]: update: now i saw u already did: https://github.com/metasfresh/metasfresh/pull/11503/files#diff-e88039b5abbd6ed8c056b87f937c1a2968fb9af8575a490da069d777e29d3716R144

Comment (metas-rc) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/excel/process/ExportToExcelProcess.java:30]: removed

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5598410_sys_gh11501_AD_Process_csv_Columns.sql (hunk 2 above)
Comment (metas-rc) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5598410_sys_gh11501_AD_Process_csv_Columns.sql:175]: Done

**Review by metas-ts — APPROVED** [review]

**Pull request merged**